### PR TITLE
Update contracts with template variables

### PR DESCRIPTION
**Pull request by Lbqds**
* update wormhole contracts
* update wormhole SDK
* add repair command for alephium

### CODE REVIEW

File: alephium/lib/wormhole.ts [hunk 1]
```diff
@@ -52,68 +60,61 @@ export class Wormhole {
...
+    private undoneSequenceTemplateVariables = {
+        undoneSequenceMaxSize: 128, // 1k
+        undoneSequenceMaxDistance: 512
     }
 
     async deployContracts(): Promise<WormholeContracts> {
         const eventEmitter = await this.deployEventEmitter()
-        const tokenWrapperFactoryDeployResult = await this.deployTokenWrapperFactory(eventEmitter.address)
+        const tokenWrapperFactoryDeployResult = await this.deployTokenWrapperFactory(eventEmitter.contractId)
+        console.log(tokenWrapperFactoryDeployResult.contractId)
```
Comment (polarker) [inline alephium/lib/wormhole.ts:111]: Is this for debugging only?

Comment (Lbqds) [reply alephium/lib/wormhole.ts:111]: Forgot to delete it, thanks.

File: sdk/js/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "build-contracts": "npm run build --prefix ../../ethereum && node scripts/copyContracts.js && typechain --target=ethers-v5 --out-dir=src/ethers-contracts contracts/*.json",
     "build-abis": "typechain --target=ethers-v5 --out-dir=src/ethers-contracts/abi src/abi/Wormhole.abi.json",
     "build-deps": "npm run build-abis && npm run build-contracts",
-    "build-lib": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && node scripts/copyEthersTypes.js && node scripts/copyWasm.js",
+    "build-lib": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && node scripts/copyEthersTypes.js && node scripts/copyWasm.js && node scripts/copyAlphArtifacts.js",
```
Comment (polarker) [inline sdk/js/package.json:16]: We could use a symbolic link: `ln -s`?

File: sdk/js/scripts/copyAlphArtifacts.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const fs = require("fs");
```
Comment (polarker) [inline sdk/js/scripts/copyAlphArtifacts.js:1]: `fs-extra` supports folder copy

**Review by polarker — COMMENTED** [review]
Great work 👍 A few minor comments, but I only review it without diving deeply

**Review by polarker — APPROVED** [review]

File: alephium/lib/wormhole.ts [hunk 4]
```diff
@@ -52,68 +60,60 @@ export class Wormhole {
...
-        )
-        return {
-            eventEmitterId: eventEmitterId,
-            tokenBridgeForChainBinCode: tokenBridgeForChain.bytecode,
-            tokenWrapperCodeHash: tokenWrapper.codeHash,
-            tokenWrapperFactoryAddress: tokenWrapperFactoryId,
-            tokenWrapperBinCode: tokenWrapper.bytecode
-        }
+    private undoneSequenceTemplateVariables = {
+        undoneSequenceMaxSize: 128, // 1k
```
[the remark this answers is not in the document]
Comment (Lbqds) [reply alephium/lib/wormhole.ts:104]: No, the type of sequence is `u64`, so the maximum size of the contract state is 1k, I will improve the comment, thanks.

File: sdk/js/package.json (hunk 2 above)
Comment (Lbqds) [reply sdk/js/package.json:16]: We need to add artifacts to the release package, I'm not too familiar with how typescript release works, so I followed the same approach as the other chains.

Comment (polarker) [reply sdk/js/package.json:16]: Sounds great then!

**Review by polarker — APPROVED** [review]
Awesome, let's merge it!

**Pull request merged**